include/nuttx/net/net.h

```diff
@@ -118,13 +118,9 @@ enum net_lltype_e
 
 typedef uint16_t sockopt_t;
 
-/* This defines the storage size of a timeout value.  This effects only
- * range of supported timeout values.  With an LSB in seciseconds, the
- * 16-bit maximum of 65535 corresponds to 1 hr 49 min 13.5 sec at decisecond
- * resolution.
- */
+/* This defines the storage size of a timeout value (seciseconds) */
 
-typedef uint16_t socktimeo_t;
+typedef int32_t socktimeo_t;
```

Comment:
@xiaoxiang781216 is there some pragmatic application of it? Is it possible in the real word a socket delay for more than 1 hr 49 min 13.5 sec ? :-)
Maybe a better option should change the resolution to accept milliseconds or even microseconds timeout value, so in this case been int32_t will make it possible to have a larger range of timeout values

Reply:
Ok, I drop this change.